lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MojoParameter.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat Inc. and others.
```

Comment:
You need to keep the copyright header of the copied file.

Reply:
Thanks, wasn't aware of the proper procedure.